ui/app/AppLayouts/Communities/views/CommunityTokenView.qml

```diff
@@ -79,6 +80,12 @@ StatusScrollView {
         id: d
 
         readonly property int iconSize: 20
+
+        property var contactDetails: null
+
+        function updateContactDetails(contactId) {
+            contactDetails = contactId !== "" ? Utils.getContactDetailsAsJson(contactId, false, true, false) : null
```

Comment:
I am setting the `getOnlineStatus` flag to true to be able to retrieve the onlineStatus of the ContactDetails. The function is defined as 

```js
function getContactDetailsAsJson(publicKey, getVerificationRequest=true, getOnlineStatus=false, includeDetails=false) {
   ...
}
```